src/posit/connect/urls_test.py

```diff
@@ -1,28 +1,35 @@
 import pytest
 
-from .urls import append_path
+from .urls import append_path, server_to_api_url, validate
 
 
-def test_fix_without_scheme():
-    with pytest.raises(ValueError):
-        append_path("foo.bar/__api__", "baz")
+def test_append_path():
+    assert append_path("http://foo.bar/__api__", "baz") == "http://foo.bar/__api__/baz"
 
 
-def test_fix_without_netloc():
-    with pytest.raises(ValueError):
-        append_path("http:///__api__", "baz")
+def test_append_path_with_leading_slash():
+    assert append_path("http://foo.bar/__api__", "/baz") == "http://foo.bar/__api__/baz"
+
+
+def test_fix_with_correct_url():
+    assert server_to_api_url("http://foo.bar/__api__") == "http://foo.bar/__api__"
+    assert server_to_api_url("http://foo.bar/__api__/") == "http://foo.bar/__api__"
```

Comment:
I noticed this case (with trailing slash) wasn't covered so I added this assertion and then made the test pass.